core/types/src/attestation_key.rs

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Attestation Key types
+
+use crate::{
+    attestation_key::QuoteSignature::{Linkable, UnLinkable},
+    new_type_accessors_impls,
+    report::{ExtendedProductId, FamilyId},
+    svn::ConfigId,
+    FfiError,
+};
+use mc_sgx_core_sys_types::{sgx_att_key_id_ext_t, sgx_ql_att_key_id_t, sgx_quote_sign_type_t};
+
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub enum MrSignerKeyHash {
+    Sha256([u8; 32]),
+    Sha384([u8; 48]),
+}
+
+#[derive(Debug, Default, Clone, Hash, PartialEq, Eq)]
+#[repr(transparent)]
+pub struct AlgorithmId(u32);
```

Comment:
The underlying values are available in `sgx_quote3.h`, but that header is in `mc-sgx-dcap-sys-types`  It may be that longer term we move all quote stuff to dcap, not sure.

Reply:
created an enum based on the underlying values, will revisit the home when the `mc-sgx-dcap-types` crate comes about